proposals/0014-expanded-comparison-sampling.md

```diff
@@ -4,8 +4,8 @@
 * Author(s): [Greg Roth](https://github.com/pow2clk),
              [Jesse Natalie](https://github.com/jenatali)
 * Sponsor: [Greg Roth](https://github.com/pow2clk)
-* Status: **Under Consideration**
-* Planned Version: Shader Model 6.8
+* Status: Complete
+* Version: Shader Model 6.8
 * Issues: [30](https://github.com/microsoft/hlsl-specs/issues/30),
```

Comment:
Should the issues listed below be removed? If not, should the relevant issues for wave size, for example, be added?